src/android/java/io/jxcore/node/BtToServerSocket.java

```diff
@@ -13,33 +13,39 @@
  */
 public class BtToServerSocket extends Thread implements StreamCopyingThread.CopyThreadCallback {
```

Comment:
Did actually move the stuff back to constructor, also made it throwing the Exception. Needed this when I got to the restructuring phase, the throwing code is now in base class.